README.md

```diff
@@ -131,15 +133,15 @@ Models should implement the following interface, implementing an `encode` functi
 
 ```python
 class MyModel():
-    def encode(self, sentences, batch_size=32, **kwargs):
+    def encode(self, sentences: list[str], **kwargs) -> list[np.ndarray] | list[torch.Tensor]:
```

Comment:
Oh this works? I always thought it had to be `typing.List` instead of `list`

Reply:
It was added in the more recent version of python. You "fix" old version of python by importing annotation from __future__.